src/java.base/share/classes/java/util/concurrent/ConcurrentHashMap.java

```diff
@@ -1999,7 +1998,7 @@ else if (f instanceof ReservationNode)
             }
         }
         if (delta != 0)
-            addCount((long)delta, binCount);
```

Comment:
Not sure why delta is not declared long as in clear().  But if not, then the explicit cast is documentation of the implementation choice to declare it int.